src/Tribe/App_Shop.php

```diff
@@ -148,19 +148,20 @@ private function get_all_products() {
 			$all_products = tribe( 'plugins.api' )->get_products();
 
 			$products = [
-				'the-events-calendar'      => (object) $all_products['the-events-calendar'],
-				'events-calendar-pro'      => (object) $all_products['events-calendar-pro'],
-				'events-virtual'           => (object) $all_products['events-virtual'],
-				'event-aggregator'         => (object) $all_products['event-aggregator'],
-				'event-automator'         => (object) $all_products['event-automator'],
-				'event-tickets'            => (object) $all_products['event-tickets'],
-				'event-tickets-plus'       => (object) $all_products['event-tickets-plus'],
-				'promoter'                 => (object) $all_products['promoter'],
-				'tribe-filterbar'          => (object) $all_products['tribe-filterbar'],
-				'events-community'         => (object) $all_products['events-community'],
-				'events-community-tickets' => (object) $all_products['events-community-tickets'],
-				'tribe-eventbrite'         => (object) $all_products['tribe-eventbrite'],
-				'image-widget-plus'        => (object) $all_products['image-widget-plus'],
+				'the-events-calendar'       => (object) $all_products['the-events-calendar'],
+				'events-calendar-pro'       => (object) $all_products['events-calendar-pro'],
+				'events-virtual'            => (object) $all_products['events-virtual'],
+				'event-aggregator'          => (object) $all_products['event-aggregator'],
+				'event-automator'           => (object) $all_products['event-automator'],
+				'event-tickets'             => (object) $all_products['event-tickets'],
+				'event-tickets-plus'        => (object) $all_products['event-tickets-plus'],
+				'promoter'                  => (object) $all_products['promoter'],
+				'tribe-filterbar'           => (object) $all_products['tribe-filterbar'],
+				'events-community'          => (object) $all_products['events-community'],
+				'events-community-tickets'  => (object) $all_products['events-community-tickets'],
+				'tribe-eventbrite'          => (object) $all_products['tribe-eventbrite'],
+				'image-widget-plus'         => (object) $all_products['image-widget-plus'],
+				'event-tickets-wallet-plus' => (object) $all_products['event-tickets-wallet-plus'],
```

Comment:
Can we add it after event tickets plus? so that they're all grouped. Sorry I didn't add this to the ticket.